ga4gh/oidcConfig.yml

```diff
@@ -0,0 +1,5 @@
+provider: "https://localhost:8443"
+-provider: "https://accounts.google.com"
+clientid: 809644742669-4rb09tba14dmivmp9r7cs966j8s13m4q.apps.googleusercontent.com
+clientsecret: 7-eoqctUk7TBjrBt_ojZ85Af
+redirecturi: https://localhost:8080/oauth2callback
```

Comment:
Actually, checked the wrong one in. I'll add oidcConfig.yml.sample which is more as you said.